Source/WebKitLegacy/mac/WebView/WebView.mm

```diff
@@ -3431,7 +3429,7 @@ - (void)_setInitiatedDrag:(BOOL)initiatedDrag
 - (void)_didCommitLoadForFrame:(WebFrame *)frame
 {
     if (frame == [self mainFrame])
-        _private->didDrawTiles = 0;
+        WTF::atomicStore(&_private->didDrawTiles, NO, std::memory_order_relaxed);
```

Comment:
I think this should use `std::memory_order_release` (or the default sequential consistent order).

Reply:
Fixed!